docs_src/microservices/device/sdk/details/CustomServiceMetrics.md

```diff
@@ -10,9 +10,9 @@ The following built-in device service metrics are collected by the Device SDK
 
 1. **EventSent** - Number of Events that have been sent from the Device Service
 2. **ReadingsSent** - Number of Reading that have been sent from the Device Service 
-3. **Common Metrics** - Set of service metrics common to all EdgeX Services. See [Common Service Metrics](../../microservices/general/#common-service-metrics) for list of all these metrics.
+3. **Common Metrics** - Set of service metrics common to all EdgeX Services. See Common Service Metrics(Link TBD) for list of all these metrics.
```

Comment:
link to this section: https://docs.edgexfoundry.org/3.1/microservices/general/#common-service-metrics

Reply:
Please make the link relative, no absolute.

Reply:
@cloudxxx8 @lenny-intel Thanks for the comment but somehow I can't find this link  https://docs.edgexfoundry.org/3.1/microservices/general/#common-service-metrics on main/napa branch

https://github.com/edgexfoundry/edgex-docs/tree/napa/docs_src/microservices/general
https://github.com/edgexfoundry/edgex-docs/tree/main/docs_src/microservices/general

Reply:
it's obviously in microservices/general/index.md#service-metrics

Reply:
found and udpated, thanks